libcontainer/configs/mount.go

```diff
@@ -1,42 +1,7 @@
 package configs
 
-import "golang.org/x/sys/unix"
```

Comment:
What is the issue with having this here? Mac is a unix, so this should work on mac, right? Why can't we share the same struct here?

Also, if Mac doesn't support bind mounts (do they really don't support it?), can't we just define that function in different platforms? Why do we need to change the struct too?

Reply:
See compilation issue,.  Indeed it doesn't support bind mounts, as well as MountAttr

Reply:
Thanks!